development_workflow.md

```diff
@@ -626,25 +626,15 @@ use a private repository in the NEST organization and do the same as Jeyashree i
 
 Abigail wants to fix a misleading piece of documentation
 --------------------------------------------------------
-Abigail is happy with the fact that other people can watch her fixing the
-documentation. She clicks the button on the GitHub page of NEST and chooses her
-personal profile as the target.
-
-She then works on her own public version of the NEST code on GitHub and creates
-a once she is done by clicking on the corresponding button.
+Abigail wants to fix the documentation of a single file and figures that changing the file directly through the GitHub web interface is the easiest way to go.
 
-She adds a comment to the Pull Request explaining the reason for the change.
+On the GitHub page of NEST she chooses her personal profile as the target and browses to the file she wants to edit. She clicks the button on the top right corner that allows her to edit this file and then fixes the documentation. Once she is done, she writes a meaningful commit message under "Commit changes" at the bottom of the page and selects the option "Create a new branch for this commit and start a pull request". She types in an appropriate branch name and finally clicks "Commit changes".
```

Comment:
I think you do not mean 'personal profile'. I think what you mean is 'personal fork'?
Something like: "On the GitHub page she navigates to her personal fork of NEST and browses [...]"